languages/elm/exercises/concept/basics/src/Greet.elm

```diff
@@ -0,0 +1,14 @@
+-- This line 'exports' the `greet` function, so that 
+-- other modules (such as the tests) can use it
+module Greet exposing (greet)
+-- or
+-- module Basics exposing (..)
```

Comment:
Still have "Basics" here. Maybe continue the previous explanation with something like `It is possible to export everything with "exposing (..)"` instead of the code example?